Code/Tools/SceneAPI/SceneCore/Containers/GraphObjectProxy.cpp

```diff
@@ -136,15 +138,19 @@ namespace AZ
             if (behaviorProperty.m_setter)
             {
                 AZStd::string_view type = FetchPythonType(*behaviorProperty.m_setter->GetArgument(1));
+		AZ_PUSH_DISABLE_WARNING_GCC("-Wdangling-pointer");
                 AzFramework::StringFunc::Append(buffer, type.data());
+		AZ_POP_DISABLE_WARNING_GCC;
             }
             AzFramework::StringFunc::Append(buffer, ")");
 
             if (behaviorProperty.m_getter)
             {
+		AZ_PUSH_DISABLE_WARNING_GCC("-Wdangling-pointer");
                 AZStd::string_view type = FetchPythonType(*behaviorProperty.m_getter->GetResult());
                 AzFramework::StringFunc::Append(buffer, "->");
                 AzFramework::StringFunc::Append(buffer, type.data());
+		AZ_POP_DISABLE_WARNING_GCC;
```

Comment:
repeat: The assignment to the AZStd::string_view `type` variable  should be changed to be of type `AZStd::string` as that is causing the dangling reference.